Formula/ripgrep.rb

```diff
@@ -14,12 +14,16 @@ class Ripgrep < Formula
 
   depends_on "asciidoc" => :build
   depends_on "docbook-xsl" => :build
+  depends_on "pcre2"
```

Comment:
This goes to the bottom of the pile now, since it isn't a `:build` dependency.

Reply:
Done